tests/Bundle/LongRunningBundle/DependencyInjection/Compiler/MonologCleanersPassBufferTest.php

```diff
@@ -6,7 +6,7 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Definition;
 
-class MonologCleanersPassBufferTest extends \PHPUnit_Framework_TestCase
+class MonologCleanersPassBufferTest extends \PHPUnit\Framework\TestCase
```

Comment:
Maybe we should move the `\PHPUnit\Framework\TestCase` items to the use `use PHPUnit\Framework\TestCase;` and only extend the `TestCase`?

Reply:
Done